services/discovery/src/app.py

```diff
@@ -21,8 +21,8 @@
 DEFAULT_OUTPUT = {
     'IsAuditAccount': False,
     'AuditCloudTrailBucketName': None,
-    'IsConnectedAccount': False,
-    'IsCloudTrailAccount': False,
+    'IsResourceOwnerAccount': False,
```

Comment:
All the changes to python below are simple find/replace to match the discovery output to the domain and top level CFN above.